redirects.next.js

```diff
@@ -215,7 +215,11 @@ module.exports = (async () => {
     '/cloud-docs/api-docs/run-tasks': '/cloud-docs/api-docs/run-tasks/run-tasks',
     '/cloud-docs/api-docs/run-tasks-integration': '/cloud-docs/api-docs/run-tasks/run-tasks-integration',
     '/language/provider-checksum-verification': '/language/files/dependency-lock#checksum-verification', // Used by the Terraform CLI to short-link to documentation.
-    '/cdktf/concepts/providers-and-resources' : '/cdktf/concepts/providers'
+    '/cdktf/concepts/providers-and-resources' : '/cdktf/concepts/providers',
+    // The /configuration pages have been deleted.
```

Comment:
_will be deleted_

Reply:
I didn't add a redirect for `/configuration` since it gets close to 0 traffic

Reply:
tbh this makes complete sense to me. Idk where we would even send folks anyway for that one